tealer/utils/analyses.py

```diff
@@ -244,7 +253,9 @@ def detect_missing_on_completion(
         if isinstance(ins, Return):
             if len(ins.prev) == 1:
                 prev = ins.prev[0]
-                if isinstance(prev, Int) and prev.value == 0:
+                if (isinstance(prev, Int) and prev.value == 0) and (
```

Comment:
I think the second `and` should be a `or` ?

We could make an internal function `is_push(ins, value)` as we are likely to see this pattern frequently